modules/carto/src/layers/carto-layer.js

```diff
@@ -6,8 +6,7 @@ const defaultProps = {
   data: null,
   credentials: null,
   onDataLoad: {type: 'function', value: tilejson => {}, compare: false},
-  // eslint-disable-next-line
-  onDataError: {type: 'function', value: err => console.error(err), compare: false}
+  onDataError: {type: 'function', value: null, compare: false}
```

Comment:
- You need `optional: true` to avoid prop type check error
- Update documentation?